### DISCUSSION TITLE
MAX vs TVM

### DISCUSSION THREAD

**Discussion by hcye2022** [modular/modular#2895] [General]
I have used TVM to deploy ML models on edge devices. It seems MAX has a lot of overlap with TVM. I am curious about whether MAX can overtake TVM in the long run, when more target platforms are added to Mojo --- would MAX match TVM features like Auto Tuning, what features can make MAX better than TVM, etc. --- general pros and cons of MAX vs TVM. Please chime in!

**Comment by Brian-M-J** [comment]
For those who don't know what TVM is: https://tvm.apache.org

In their LLVM talk, the team mentioned ML compilers in general. You can see what they said [here](https://youtu.be/SEwTjZvy8vw?t=267).

Other than that, I don't have the necessary technical knowledge for any more comparisons.
